svthalia/concrexit
# Remove all remaining usage of the MultiLanguageField

Closes #1200

Remove all remaining usage of the MultiLanguageField and the language property.

### How to test
Steps to test the changes you made:
1. Check that all push notifications are still created correctly
2. Check that the newsletter still works correctly
3. Maybe do some manual checks if I missed something by looking for `_en` or `language`
